scripts/th/_th-version.sh

```diff
@@ -15,46 +15,133 @@
  # See the License for the specific language governing permissions and
  # limitations under the License.
 
-ROOT_DIR=$(realpath $(dirname "$0")/../..)
-cd $ROOT_DIR
-
-SDK_SHA=$(cat $ROOT_DIR/backend/app/core/config.py | grep SDK_SHA | cut -d'"' -f 2 | cut -d"'" -f 2)
-SDK_DOCKER_IMAGE=$(cat $ROOT_DIR/backend/app/core/config.py | grep SDK_DOCKER_IMAGE | cut -d'"' -f 2 | cut -d"'" -f 2)
-SDK_DOCKER_TAG=$(cat $ROOT_DIR/backend/app/core/config.py | grep SDK_DOCKER_TAG | cut -d'"' -f 2 | cut -d"'" -f 2)
+# Containers
+BACKEND_CONTAINER_NAME="certification-tool-backend-1"
+FRONTEND_CONTAINER_NAME="certification-tool-frontend-1"
+DB_CONTAINER_NAME="certification-tool-db-1"
+PROXY_CONTAINER_NAME="certification-tool-proxy-1"
 
 read_version() {
-    # Input validation for path
-    if [ -z "$1" ]; then
-        echo "Please provide a path."
-        return 1
-    fi
-
-    # Input validation for title
-    if [ -z "$2" ]; then
-        echo "Please provide a title."
-        return 1
-    fi
-
     file_path="$1"
     title="$2"
 
-    # Check if file exists
-    if [ ! -f "$file_path" ]; then
-        echo "File not found!"
+    # Check if the container is running
+    if ! docker ps --format '{{.Names}}' | grep -q "^${BACKEND_CONTAINER_NAME}$"; then
+        echo "$title: The '$BACKEND_CONTAINER_NAME' container it not running."
+        return
+    fi
+
+    # Check if file exists in the container
+    if ! docker exec "$BACKEND_CONTAINER_NAME" sh -c "[ -f $file_path ]"; then
+        echo "File '$file_path' not found in the container!"
         return 1
     fi
 
-    # Read the file content into a variable
-    file_content=$(cat "$file_path")
+    # Read the file content from the container into a variable
+    file_content=$(docker exec "$BACKEND_CONTAINER_NAME" cat "$file_path")
 
     # Print the title and file content
     echo "$title: $file_content"
+}
+
+show_backend_info() {
+    # Check if the container is running
+    if ! docker ps --format '{{.Names}}' | grep -q "^${BACKEND_CONTAINER_NAME}$"; then
+        echo
+        echo "$BACKEND_CONTAINER_NAME"
+        echo "     The container is not running."
+        return
+    fi
 
-    cd $ROOT_DIR
+    # Get info
+    FILE_PATH="app/core/config.py"
```

Comment:
Matter's SDK SHA information its not stored in this file anymore, it's on `test_collections/matter/config.py`

Reply:
The test collections folder appears to be empty
```
root@68e91f928bb4:/app/test_collections# ls -l
total 0
root@68e91f928bb4:/app/test_collections#
```

Does anything need to be run?

Reply:
> Matter's SDK SHA information its not stored in this file anymore, it's on `test_collections/matter/config.py`

This change is only for TH version v2.11 and onwards.

Reply:
I understand now, I'm on 2.10+

I will make it check on `test_collections/matter/config.py` first, if not available, then on `app/core/config.py` to make it backwards compatible